components/formFields.js

```diff
@@ -38,34 +38,82 @@ export const CheckBoxFormField = (props) => {
   );
 };
 
-export const CustomCheckBoxContainerParent = (props) => {
+export const LabelListContainer = (props) => {
   return (
-    <View style={styles.customCheckBoxContainerParent}>
+    <View style={styles.customCheckBoxContainer}>
       <TextSourceSans style={styles.formLabel}>{props.field}</TextSourceSans>
       {props.children}
       {props.children.touched && <TextSourceSans style={styles.formError}>{props.error}</TextSourceSans>}
     </View>
   );
 };
 
-export const CustomCheckBoxFormField = (props) => {
-  const plusIcon = (
-    <IconFA name='plus' size={16} color={COLORS.text.main} />
-  );
+export const LabelList = props => {
```

Comment:
I think this can go into it's own component as it's now so specific, I started a general form field file for reuseable, basic form fields but I think this is now past that, what do you think?

Reply:
yes agree!